stories/dooboo-ui/Accordion.stories.tsx

```diff
@@ -29,28 +46,34 @@ const Default = (): React.ReactElement => {
       body: 'Hi. I love this component. What do you think?',
     },
     {
-      title: 'See this one too',
-      body: 'Yes. You can have more items.',
+      title: 'Title 2',
+      body: 'Hi. I love this component. What do you think?',
     },
     {
-      title: 'Thrid thing',
-      body: `What about very long text? What about very long text?
-        What about very long text? What about very long text?
-        What about very long text? What about very long text?
-        What about very long text? What about very long text?
-        What about very long text? What about very long text?
-        What about very long text? What about very long text?`,
+      title: 'Title 3',
+      body:
+      'Hi. I love this component. What do you think?',
```

Comment:
What if changing some texts for example? Like a very long text?

Reply:
> What if changing some texts for example? Like a very long text?

Okay. I will add some more examples.